### DISCUSSION TITLE
why `jotai` library cannot be auto-imported ?

### DISCUSSION THREAD

**Discussion by MarkShawn2020** [pmndrs/jotai#2428] [Q&A] [answered]
I use Webstorm to develop react apps, and recently I am giving `jotai` a shot as a complement of `zustand`.

But I happed to find that I cannot auto import the npm library of `jotai`.

Suppose I wrote `export const conversationIdAtom = atom(0)`, the `atom` would be red since IDE doesn't know it, and I mouse over it, press `⌘ + Enter`, it is expected to see the import hints but not, how?

<img width="831" alt="image" src="https://github.com/pmndrs/jotai/assets/33591398/930622ed-c369-4879-9174-7d0702f5b804">

---

Note: 
1. other packages are ok, such as `zustand`.
2. vscode also fails at this target.

**Comment by dai-shi** [comment]
> other packages are ok, such as zustand.

How about `valtio`?

**Comment by himself65** [comment]
<img width="524" alt="image" src="https://github.com/pmndrs/jotai/assets/14026360/3c4b0fbd-1661-43b7-af43-84dfb8df6100">

Right now, I use jotai in WebStorm; there must be some misconfig in your environment.

**Reply by MarkShawn2020** [reply]
so, what's you config of your environemnt?
or what do you think is the most important ones?

**Comment by MarkShawn2020** [comment, marked as answer]
since currently my webstorm seems working now, I decides to close this issue.

<img width="509" alt="image" src="https://github.com/pmndrs/jotai/assets/33591398/99218305-f3a1-4416-b72a-4b53f1be7b6a">

**Reply by himself65** [reply]
I think so
